### PR TITLE
fix gruntfile

### PR DESCRIPTION
**Pull request by samogot**
- replace beautifier with eslint:fix in main task
- remove all mentions of old unused screeps-visual

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 300/502
    Coefficient: 0.597609561753
    Merging in 4 days 0 hours
    Age 0 days 0 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 305/502
    Coefficient: 0.607569721116
    Merging in 3 days 6 hours
    Age 0 days 15 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 332/502
    Coefficient: 0.661354581673
    Merging in 2 days 10 hours
    Age 0 days 22 hours

File: Gruntfile.js [hunk 1]
```diff
@@ -236,12 +226,12 @@ module.exports = function(grunt) {
     },
   });
 
-  grunt.registerTask('default', ['jshint', 'jsbeautifier', 'jscs', 'clean', 'copy:uglify', 'copy:main', 'copy:profiler', 'copy:visualizer', 'screeps']);
+  grunt.registerTask('default', ['jshint', 'jsbeautifier', 'jscs', 'clean', 'copy:uglify', 'copy:main', 'copy:profiler', 'screeps']);
```
Comment (sparr) [inline Gruntfile.js]: still using jsbeautifier here, as well as on the release task?

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 393/502
    Coefficient: 0.782868525896
    Merging in 0 days 23 hours
    Age 1 days 4 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 397/506
    Coefficient: 0.784584980237
    Merging in 1 days 17 hours
    Age 1 days 12 hours

**Pull request merged**